Resources/Audio/Ambience/Antag/attributions.yml

```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
+- files:
+  lisence: "CC0 1.0 Universal"
```

Comment:
license spelled wrong + its CC0-1.0

Reply:
i copy pasted it from the artists github but i will change that

Reply:
license as in the word license you spelled it lisence :trollface: